.github/workflows/deploy.yml

```diff
@@ -93,3 +87,71 @@ jobs:
           SLACK_COLOR: ${{job.status}}
           SLACK_FOOTER: ""
           MSG_MINIMAL: actions url,commit,ref
+  cypress-test:
+    name: Cypress Tests
+    needs: deploy
+    if: ${{ github.ref != 'refs/heads/prod' }}
```

Comment:
@aalousi why do we not run this for prod?
Thanks!

Reply:
Typically, we do not run in prod because we cant add data in Prod or delete it. Also, we dont have specific accesses for Prod. therefore, we prevent running our automated regressions in prod.

Reply:
thanks that makes sense and I guessed it was something along those lines, resolving this.